games-board/cockatrice/cockatrice-20230914.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit xdg cmake
+
+MY_PV="${PV:0:4}-${PV:4:2}-${PV:6:2}-Release-2.9.0"
+
+DESCRIPTION="Open-source multiplatform software for playing card games over a network"
+HOMEPAGE="https://github.com/Cockatrice/Cockatrice"
+SRC_URI="https://github.com/Cockatrice/Cockatrice/archive/refs/tags/${MY_PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/Cockatrice-${MY_PV}"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+client +oracle test server"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+    dev-qt/qtcore:5
+    dev-qt/qtnetwork:5
+    dev-qt/qtwidgets:5
+    client? (
+        dev-qt/qtconcurrent:5
+        dev-qt/qtgui:5
+        dev-qt/qtmultimedia:5
+        dev-qt/qtprintsupport:5
+        dev-qt/qtsvg:5
+        dev-qt/qtwebsockets:5
+    )
+    oracle? (
+        dev-qt/qtconcurrent:5
+        dev-qt/qtsvg:5
+        sys-libs/zlib
+        app-arch/xz-utils
+    )
+    server? (
+        dev-qt/qtsql:5
+        dev-qt/qtwebsockets:5
+        dev-db/mysql-connector-c
+    )"
+DEPEND="
+    ${RDEPEND}
+    test? ( dev-cpp/gtest )"
+BDEPEND="
+    dev-qt/linguist-tools:5
+    dev-libs/protobuf"
```

Comment:
protobuf is built against as well, so its not just BDEPEND. so add it to RDEPEND/DEPEND as well.

Reply:
Ok I've added protobuf in DEPEND